<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="contrato_tenant-20240116180709" tests="1" file=".py" time="97.864" timestamp="2024-01-16T18:08:47" failures="1" errors="0" skipped="0">
	<testcase classname="contrato_tenant" name="test_contrato_tenant" time="97.864" timestamp="2024-01-16T18:08:47" file="contrato.py" line="22">
		<failure type="AssertionError" message=""><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\contrato.py", line 96, in test_contrato_tenant
    validate_text_visible(self.driver, element1, number_expected)
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Elements.py", line 274, in validate_text_visible
    assert text_obtained == text_expected
AssertionError
]]></failure>
		<system-out><![CDATA[¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado 3 veces con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado 1 veces con éxito!
¡Elemento encontrado y clickeado con éxito!
¡Elemento encontrado y clickeado con éxito!
El texto encontrado es  Estado De Mis Contratos
¡Elemento encontrado y clickeado con éxito!
El texto es visible para el usuario
]]></system-out>
	</testcase>
</testsuite>
